templates/README.md

```diff
@@ -0,0 +1,36 @@
+This package contains simple project templates for the GeoBlazor package. It is designed to allow users to easily create a new project with the GeoBlazor package installed and configured.
+
+There are currently three templates available:
+
+Web App (Server-side Blazor)
+
+Web Assembly App (Client-side Blazor)
+
+Maui Hybrid Blazor App
+
+
+to install this template package run:
+```
+dotnet new install GeoBlazor.Templates
+```
+
+to create a new Web App project run:
+```
+dotnet new geoblazortemplateserver -p <project name>
```

Comment:
suggested naming improvements for templates (shorter and easier to read):
- geoblazor-server
- geoblazor-wasm
- geoblazor-maui

Reply:
I updated them, but I liked template in the name to not confuse them with the default package